src/nupic/types/ClassifierResult.hpp

```diff
@@ -33,6 +33,7 @@ namespace nupic {
 namespace types {
 
 using namespace std;
+using T = vector<Real64>;
```

Comment:
Question: Is there a significant difference between this and a typedef?   I've never seen this syntax before.  I'm just curious, no need to change it.

Nitpick: Either give it a more descriptive name or add a comment.
Maybe name it `PDF` for probability density function?

Reply:
Here it's the same, `using T = ` is a newer, c++11+ construct. 
https://stackoverflow.com/questions/10747810/what-is-the-difference-between-typedef-and-using-in-c11 

> Maybe name it PDF for probability density function? 

good point, I didn't like the T either